### ISSUE TITLE
make jkan social?

### ISSUE THREAD

**Issue by iltempe** [timwis/jkan#112]
what do you think to make Jkan more social? I mean include the possibility to create community, comments the datasets published. is it a crazy thing?

**Comment by jalbertbowden** [comment]
isn't there a plan to implement disqus? i think this is a novel idea, especially allowing signup/in via 3rd party like github/stackoverflow/etc.

**Comment by iltempe** [comment]
yes! disquis should be an idea, I mean, my portal is intended for "normal people" that are not part of governements staff so social should be nice.

**Comment by JJediny** [comment]
There was an issue #5 for adding disqus ([which can certainly be done](https://github.com/tomjohnson1492/documentation-theme-jekyll/search?utf8=%E2%9C%93&q=disqus)) but there is also already a setting to turn on [hypothesis](https://hypothes.is/) which not only also for threaded conversation but annotation of any content on the site, just change the `hypothesis` setting in the `_config.yml` to true:

https://github.com/timwis/jkan/blob/gh-pages/_config.yml#L66

**Comment by timwis** [comment]
I think adding disqus comments would be a great addition. It also seems pretty straightforward to do. Would anyone like to give it a shot?

**Comment by iltempe** [comment]
I've added the possibility to make comments with DISQUS on datasets.

It's just a question to set a tag in the dataset md file, "comment: true" and in _config.yml your disqus shortname.

In dataset.html page if inserted the JS code to enable disqus where comments are enable.

you can look everythig on http://iltempe.github.io/opendatagentediprato/datasets/

you have a pull request on jkan, please verify wich changes you need.

**Comment by willemvanopstal** [comment]
@iltempe Looks good with the disquss comment sections. 
Can you explain a little bit more on how to incorporate this functionaility?

**Comment by MatheusRV** [comment]
https://staticman.net has a good approuch about this.. take a look.

**Comment by timwis** [comment]
Wow that's a pretty cool approach! Feels more jekylly than something like disqus would, which is nice.
